### ISSUE TITLE
Permalink Manager for qTranslate-XT

### ISSUE THREAD

**Issue by eirikvold** [qtranslate/qtranslate-xt#847]
Hi 
Since the issue about the importance of slugs qTranslate-XT
I have been in contact with author of Permalink Manager plugin
Maybe this could result in something great?

https://wordpress.org/support/topic/qtranslate-xt-extended-support/#new-topic-0

**Comment by bagaweb** [comment]
That looks interesting, thanks @eirikvold.

**Comment by herrvigg** [comment]
Yes sure, seen that stats Permalink Manager looks like a good plugin.

Maybe that would be a more sustainable solution because the permalinks are quite a specific functionality not directly necessary in qTranslate but i understand well the need. If the plugin does some of the job we could see how to integrate with it instead.

In that case maybe better to postpone the integration of #671 to avoid too many back and forth moves. It would be confusing if we add the legacy slug support and remove it later.

**Comment by eirikvold** [comment]
Cool. 
Seems that the Plugin Author of Permalink Manager also was open to take a look at it. Maybe it's a good idea to reach out to him @herrvigg

**Comment by moskalukigor** [comment]
Plugin author [made](https://wordpress.org/support/topic/no-working-with-q-translate-xt-2/) the integration snippet:

https://gist.github.com/maciejbis/f93b14c89cb74c965a50eb4e30316f03
